### PR TITLE
fix(kit): `InputPhoneInternational` fix chevron

### PR DESCRIPTION
**Pull request by waterplea**
Closes #

### CODE REVIEW

File: projects/legacy/components/arrow/arrow.component.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import {
 import {TUI_ARROW_OPTIONS} from './arrow.options';
 
 /**
- * @deprecated: drop in v5.0
+ * @deprecated: drop in v5.0 use {@link TuiChevron}
```
Comment (nsbarsukov) [inline projects/legacy/components/arrow/arrow.component.ts:18]: Should we add link to this example ?

https://taiga-ui.dev/components/textfield#dropdown

File: projects/legacy/components/input-phone-international/input-phone-international.component.ts [hunk 2]
```diff
@@ -56,7 +56,7 @@ import {tuiExtractValueFromEvent} from './utils/extract-value-from-event';
 const MASK_SYMBOLS = /[ \-_()]/g;
 
 /**
- * @deprecated: drop in v5.0
+ * @deprecated: drop in v5.0 use {@link TuiInputPhoneInternational}
```
Comment (nsbarsukov) [inline projects/legacy/components/input-phone-international/input-phone-international.component.ts:59]:
```suggestion
 * @deprecated: drop in v5.0 use {@link TuiInputPhoneInternational} from @taiga-ui/kit
```

\+ add link to https://taiga-ui.dev/components/input-phone-international

File: projects/legacy/components/sheet/sheet-tokens.ts [hunk 3]
```diff
@@ -4,22 +4,19 @@ import type {Observable} from 'rxjs';
 import type {TuiSheetRequiredProps} from './sheet';
 
 /**
- * @deprecated: drop in v5.0
- * Sheet main component
+ * @deprecated: drop in v5.0 use {@link TuiSheetDialog}
```
Comment (nsbarsukov) [inline projects/legacy/components/sheet/sheet-tokens.ts:7]: Should we also add link to https://taiga-ui.dev/components/sheet-dialog ?

File: projects/legacy/components/svg/content-processor.ts [hunk 4]
```diff
@@ -14,7 +14,7 @@ function extractLinearGradientIdsFromSvg(svg: string): string[] {
 }
 
 /**
- * @deprecated: drop in v5.0
+ * @deprecated: drop in v5.0 use {@link TuiIcon}
```
Comment (nsbarsukov) [inline projects/legacy/components/svg/content-processor.ts:17]: https://taiga-ui.dev/components/icon ?

Comment (nsbarsukov) [reply projects/legacy/components/svg/content-processor.ts:17]: (and all other same cases in this PR)

**Review by nsbarsukov — APPROVED** [review]

**Review by splincode — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**